db/schema.rb

```diff
@@ -71,7 +71,7 @@
   add_index "goods", ["journal_id"], name: "index_goods_on_journal_id", using: :btree
   add_index "goods", ["user_id"], name: "index_goods_on_user_id", using: :btree
 
-  create_table "journals", force: true do |t|
+  create_table "journals", force: :cascade do |t|
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.